### ISSUE TITLE
[GENERAL SUPPORT]:

### ISSUE THREAD

**Issue by vedadb** [facebook/Ax#5237]
### Question

Hi, thanks for the great work with Ax and BoTorch!

I have a current use-case which takes n1 input parameters but is evaluated on an irregular grid with n2 points i.e. the output is

x1,y1,val1
x2,y2,val2
...
x_n2,y_n2,val_n2

I want to add the x and y as input parameters to my GP model (as they are spatially related) in addition to my n1 input parameters, as I can observe that the model generalizes better with that information in hand.

The question is if I can somehow re-define my acquisition function to evaluate it at those points (x1,y2)-(xn,yn) and minimize a derived value say standard deviation (or standard error, or average value) of [val1,val2,..., val_n2]

### Please provide any relevant code snippet if applicable.

```shell

```

### Code of Conduct

- [x] I agree to follow this Ax's Code of Conduct

**Comment by saitcakmak** [comment]
Hi @vedadb. I am not sure if I fully understand the problem you are describing, so I'll paraphrase.

- Your search space is parameters x, y
- The metric / objective function produces val, for a given x, y pair.
- You have n1 points in your training data, x_i, y_i evaluated, producing val_i (n1 such pairs)
- You want to fit a GP to this training dataset of n1 observations
- You have n2 other points (x, y) that you want to predict the corresponding val using the GP.

This part so far is quite easily doable using Ax. You can create an experiment, attach your training data, then predict the GP predictions on the new points. 

I am not fully clear about the proposed acquisition function here, but you can easily define one in BoTorch and used it within Ax (see https://ax.dev/docs/next/tutorials/modular_botorch/). The structure you describe reminds me of some of the active learning acquisition functions like https://github.com/meta-pytorch/botorch/blob/main/botorch/acquisition/active_learning.py#L46
